lib/models/addon.js

```diff
@@ -175,6 +175,15 @@ function registryHasPreprocessor(registry, type) {
   @param {Project} project The current project (deprecated)
 */
 let addonProto = {
+
+  /**
+  Initializes the addon.  If you override this method make sure and call `this._super()` or your addon will not work.
+
+  @public
+  @method init
+  @param {Project|Addon} parent The project or addon that directly depends on this addon
+  @param {Project} project The current project (deprecated)
```

Comment:
I think argument destructuring is not available on Node 4 though, and since we don't run Ember CLI itself through Babel we can't use it yet.